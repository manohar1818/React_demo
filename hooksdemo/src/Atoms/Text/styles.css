.text {
  font-size: 14px;
  font-family: sans-serif;
  margin: 0;
}

.text.paragraph {
  font-size: 14px;
  color: #808080;
}

.text.subheading {
  font-size: 18px;
  font-weight: bold;
}

.text.heading {
  font-size: 24px;
  font-weight: bold;
}

.text.gutterBottom {
  margin-bottom: 0.5em;
}
