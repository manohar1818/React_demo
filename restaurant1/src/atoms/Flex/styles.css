.flex {
  display: flex;
  flex-wrap: wrap;
}

.flex-item {
  flex: 0.001;
}

@media screen and (max-width: 600px) {
  .flex-item {
    flex-basis: 100%;
  }
}

@media screen and (min-width: 601px) and (max-width: 1000px) {
  .flex-item {
    flex-basis: 50%;
  }
}

@media screen and (min-width: 1001px) {
  .flex-item {
    flex-basis: 25%;
  }
}
