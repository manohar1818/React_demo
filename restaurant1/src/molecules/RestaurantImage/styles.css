.restaurantImage.container {
  position: relative;
  display: flex;
  justify-content: center;
  overflow: hidden;
}

.restaurantImage.container button {
  position: absolute;
  align-self: center;
  color: white;
  border: none;
  background: #f44336;
  padding: 0.8em 1.3em;
}

.restaurantImage.container img {
  width: 100%;
  height: 100%;
  transform: scale(1);
  transition: all 0.3s;
}

.restaurantImage.container:hover img {
  transform: scale(1.1);
  transition: all 0.3s;
}

.deliveryDuration {
  position: absolute;
  right: 0;
  bottom: 0;
  background: white;
  padding: 0.1em 0.5em;
}

.rightLabelsGroup {
  position: absolute;
  top: 0;
  right: 0;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  max-height: 80%;
}

.rightLabel {
  color: white;
  background: #ea5b31;
  font-family: sans-serif;
  text-align: center;
  padding: 0.3em 0.7em;
  margin-top: 1em;
}

.leftLabelsGroup {
  position: absolute;
  top: 0;
  left: 0;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  max-height: 80%;
}

.leftLabel {
  color: white;
  background: rgba(0, 0, 0, 0.5);
  font-family: sans-serif;
  text-align: center;
  padding: 0.3em 0.7em;
  margin-top: 1em;
}
