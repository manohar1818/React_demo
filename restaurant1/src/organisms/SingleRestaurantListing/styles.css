.single-restaurant-listing.padder {
  padding: 0em 0.5em 3em 0.5em;
}
