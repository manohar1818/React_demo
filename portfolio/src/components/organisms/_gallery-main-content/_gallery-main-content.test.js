import React from 'react';
import { create } from "react-test-renderer";
import { render ,fireEvent} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import GalleryMain from './_gallery-main-content';
import { makeStyles } from "@material-ui/core/styles";

describe("GalleryMain component",() =>{
    test("Matches the snapshot",() => {
        const galleryMain= create(<GalleryMain/>);
        expect(galleryMain.toJSON()).toMatchSnapshot();

    })
    // test("GalleryMain is rendered",() => {
    //     const { getByTestId, getByText } = render(<GalleryMain/>);
    //     expect(getByTestId("galleryMain")).toBeInTheDocument();
        
    // });
   
   
   

});