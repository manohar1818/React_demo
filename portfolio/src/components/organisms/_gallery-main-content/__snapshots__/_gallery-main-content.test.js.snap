// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryMain component Matches the snapshot 1`] = `
<div
  className="makeStyles-styleGalleryMainContent-1"
  data-testid="galleryMain"
>
  <div
    className="makeStyles-styleGrid-2"
  >
    <div
      className="makeStyles-styleImage-3"
      data-testid="image"
    >
      <div
        className="makeStyles-picture-4"
        style={
          Object {
            "backgroundImage": "URL(https://www.birmingham.ac.uk/Images/News/Forest-900.jpg)",
          }
        }
      />
    </div>
    <div
      className="makeStyles-styleImage-3"
      data-testid="image"
    >
      <div
        className="makeStyles-picture-4"
        style={
          Object {
            "backgroundImage": "URL(https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQIWGOJcO5wb_qh4welWgFUgsZPHXMetI-iVg&usqp=CAU)",
          }
        }
      />
    </div>
    <div
      className="makeStyles-styleImage-3"
      data-testid="image"
    >
      <div
        className="makeStyles-picture-4"
        style={
          Object {
            "backgroundImage": "URL(https://miro.medium.com/max/8576/0*zBu6EBAwjXXXHz-z)",
          }
        }
      />
    </div>
    <div
      className="makeStyles-styleImage-3"
      data-testid="image"
    >
      <div
        className="makeStyles-picture-4"
        style={
          Object {
            "backgroundImage": "URL(https://api.timeforkids.com/wp-content/uploads/2019/09/final-cover-forest.jpg)",
          }
        }
      />
    </div>
    <div
      className="makeStyles-styleImage-3"
      data-testid="image"
    >
      <div
        className="makeStyles-picture-4"
        style={
          Object {
            "backgroundImage": "URL(https://www.orfonline.org/wp-content/uploads/2020/03/India-forests-1280x720.jpg)",
          }
        }
      />
    </div>
    <div
      className="makeStyles-styleImage-3"
      data-testid="image"
    >
      <div
        className="makeStyles-picture-4"
        style={
          Object {
            "backgroundImage": "URL(https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQfbbQr8RYShw7zga6H2q9kTF78VpUMq22VGA&usqp=CAU)",
          }
        }
      />
    </div>
  </div>
</div>
`;
